airflow/utils/__init__.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from __future__ import absolute_import
+
+import warnings
+
+from .decorators import apply_defaults as _apply_defaults
```

Comment:
@aeon : I see your Python mailing list, and I raise you a [PEP](https://www.python.org/dev/peps/pep-0328/). If the BDFL says it is ok, it's fine by me. Here, I want to make sure I import from the decorators file and not any other package that might name clash.